app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(human["name"], human["age"]) for human in people]
+
+    for human in people:
+        if human.get("wife"):
+            Person.people[human["name"]].wife\
+                = Person.people[human["wife"]]
+        elif human.get("husband"):
+            Person.people[human["name"]].husband\
+                = Person.people[human["husband"]]
```

Comment:
The code attempts to set `wife` and `husband` attributes on `Person` instances, but these attributes are not defined in the `Person` class. This will result in an `AttributeError`. Please define these attributes in the `Person` class.